src/mixins/nft.js

```diff
@@ -297,6 +297,26 @@ export default {
       this.updateDisplayNameList([...new Set(array)]);
       this.isHistoryInfoLoading = false;
     },
+    async getNFTEventsAll() {
+      let data;
+      let nextKey;
+      let count;
+      let events = [];
+      do {
+        // eslint-disable-next-line no-await-in-loop
+        ({ data } = await this.$api.get(
+          getNFTEvents({
+            classId: this.classId,
+            key: nextKey,
+            limit: NFT_INDEXER_LIMIT_MAX,
+          })
+        ));
+        nextKey = data.pagination.next_key;
+        ({ count } = data.pagination);
+        events = events.concat(data.events);
```

Comment:
`events` can be a `const`
```suggestion
        events.push(...data.events);
```